common/src/main/java/com/sequenceiq/cloudbreak/util/FreeIpaPasswordUtil.java

```diff
@@ -24,8 +26,9 @@ private FreeIpaPasswordUtil() {
     public static String generatePassword() {
         String upperCaseLetters = RandomStringUtils.random(PWD_PART_LENGTH, UPPER_CASE_LETTERS_START, UPPER_CASE_LETTERS_END, true, true);
         String lowerCaseLetters = RandomStringUtils.random(PWD_PART_LENGTH, LOWER_CASE_LETTERS_START, LOWER_CASE_LETTERS_END, true, true);
```

Comment:
`randomAlphabetic(PWD_PART_LENGTH).toUpperCase()/.toLowerCase()` would be more readable I think